pages/common/routers/form.js

```diff
@@ -41,11 +41,10 @@ module.exports = ({
 
   const _configure = (req, res, next) => {
     req.form = req.form || {};
-    req.model = req.model || {};
-    req.model.id = req.model.id || `new-${model}`;
     req.form.schema = schema;
     req.session.form = req.session.form || {};
     req.session.form[req.model.id] = req.session.form[req.model.id] || {};
+    req.session.form[req.model.id].values = req.session.form[req.model.id].values || {};
```

Comment:
We might get some value out of the `set` method from lodash here.

```
set(req.session, `form.${req.model.id}.values`, {});
```

This will replace the previous three lines (I think).

Not one for _right now_, but worth considering when we're setting deep properties on things that may or may not exist.

Reply:
yeah that's a good shout - I didn't know about `set`

Reply:
Yesterday me says "same".

Yesterday me also wonders how I'd managed to merrily use `get` for so long without wondering if `set` also existed.